content/posts/easystats_threeyears.Rmd

```diff
@@ -0,0 +1,144 @@
+
+- [**datawizard**](https://easystats.github.io/datawizard): 🧙 The most recent addition to the ecosystem! This package form the data wrangling backend for the entire ecosystem and can be just what you are looking for if you want to do *tidyverse*-like data wrangling but wish to avoid adding dependencies to your project.
+
+We then have specialized packages to extract additional information from statistical models.
+
+- [**bayestestR**](https://easystats.github.io/bayestestR): 👻 A package that provides utility tools for manipulating and visualizing Bayesian statistical models.
+
+- [**effectsize**](https://easystats.github.io/effectsize): 🐉 A one-stop solution for computing pretty much every effect size known to the mankind, well, almost every. 😅
+
+- [**correlation**](https://easystats.github.io/correlation): 🔗 A package dedicated for computing and visualizing probably the most comprehensive suite of correlations.
+
+- [**modelbased**](https://easystats.github.io/modelbased): 📈 A utility package to work with predictions from statistical models, and can be used to compute and visualize marginal means or contrast analyses.
+
+The information about statistical models are extracted and aggregated in the following two packages:
+
+- [**performance**](https://easystats.github.io/performance): 💪 Computes, analyzes, and tests the performance of statistical model(s).
+
+- [**parameters**](https://easystats.github.io/parameters): 📊 Extracts a comprehensive dataframe of almost any statistical model's parameters and provides helpers to present them in elegant tables and plots.
+
+Finally, the two high-level packages utilize other packages to communicate results from statistical analyses, either graphically or textually.
+
+- [**see**](https://easystats.github.io/see): 🎨 This package forms the visualization backend for the entire ecosystem and interfaces with *ggplot2* to support plotting methods for various *easystats* objects.
+
+- [**report**](https://easystats.github.io/report): 📜 🎉 This packages provides an automated way to create text reports detailing results from statistical analyses.
+
+Except for *datawizard*, *report*, and *modelbased* packages, which are still in their teenage years, the rest of the packages are legally permitted to drive your statistical analyse as they have been published in peer-reviewed journals^[You can use `citation()` function to see reference for each package.] and enjoy a regular health checkup with a large suite of unit tests.
+
+### Soaring Like an Eagle!
+
+To our utmost delight, these packages have already been downloaded **over 6 million times**! 🤯
+
+It has been an overwhelming experience to see the warm welcome this project has received in the `#rstats` community on Twitter and LinkedIn.
+
+### "Let brotherly love continue"
+
+Not only have the packages gained popularity among users, but also fellow R developers have started to rely on *easystats* packages for their own project.
```

Comment:
I wonder if we could give examples of popular packages that rely on us, so that readers could be like "hey I use that package I don't know that was thanks to easystats too!" FOR INSTANCE I know people that were impressed when they learned that the famous ggstatsplot was relying on it :p

Reply:
Sure, sounds good to me 😅 

I sometimes worry that we typically just list _only_ the packages from team members (_ggstatsplot_, _sjPlot,_ _marginaleffects,_ etc.), which makes it seem like the other developers we are talking about is none other than us 🤣

Reply:
that's a good point haha is there a way to easily query all our reverse-dependencies and sort it by like number of downloads or something?

Reply:
though I suspect that the vast majority of users don't know who the authors of packages are, like I had students and colleagues using mine for AGES before one of them realized that I was involved - and when I started R I didn't care about the names either especially when you can't put faces to them

Reply:
Yes, but this section of the post is targeting the _developers_, and developers _do_ know such details, even if users don't. Thus, my worry.

Better to just leave this sentence as it is, IMHO.

Reply:
dotwhisker and effects are surely better known packages

Reply:
> is he officially? and if not, why not?

Because then we get to count 2 more packages as "external"? j/k